CHANGELOG.md

```diff
@@ -28,6 +26,29 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 - Catalog Crawler CE: `ghcr.io/sovity/catalog-crawler-ce:{{ CE VERSION }}`
 - Sovity EDC CE: {{ CE Release Link }}
 
+## [v4.1.0] - 2024-09-04
+
+### Overview
+
+Quality of Life improvements
+
+### Detailed Changes
+
+#### Minor
+
+- Added auto refresh for pages with connectors every 30 seconds
+
+### Deployment Migration Notes
```

Comment:
Shouldn't the sections go away when they are empty?
If not, then I need to fix the changelog in the EDC CE/EE